src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { SelectOption } from '../SelectOption/SelectOption';
+import { TextField } from '../TextField/TextField';
+
+interface Props {
+  onChangeTodo: (title: string, userId: number, completed?: boolean) => void;
+}
+
+export const NewTodo: React.FC<Props> = ({ onChangeTodo }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number>(0);
+  const [submited, setSubmited] = useState<boolean>(false);
```

Comment:
If u use a `simple type` as the `initial value`, the `typescript` itself understands what type should be in the state. This is redundant
```suggestion
  const [title, setTitle] = useState('');
  const [selectedUser, setSelectedUser] = useState(0);
  const [submited, setSubmited] = useState(false);
```